### ISSUE TITLE
devcontainer not working. Linked Repository archived 2024-12-12

### ISSUE THREAD

**Issue by picpromusic** [pulumi/pulumi-provider-boilerplate#103]
### What happened?

If i try to build a new provider with this template using the provided devcontainer (1 year old) it does not work.
The linked repository for the devcontainer definition is archived 2024-12-12 and it seems to contain a recursive definition of the docker-images for ghcr.io/pulumi/devcontainer:extra.

It doesn't seem to be an easy fix. Maybe the archive of https://github.com/pulumi/devcontainer is another sign that this functionality is broken. 

It would be best to repair it. But if not possible it would be better to delete the files and hints in the README and create a feature-request. 

### Example

#git clone
#start devcontainer
#go version prints 1.20 which is to old
or start
make build install

### Output of `pulumi about`

none

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mjeffryes** [comment]
Yes, unfortunately the devcontainer support has not been maintained. We're in process on an update to this boilerplate to pick up our latest workflows and tools. I'll ping this ticket when that rolls out.
